src/analysis/Environments/Environments.ts

```diff
@@ -210,10 +208,6 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
 
   useEffect(() => {
     loadData();
-    const interval = setInterval(refreshData, 30000);
```

Comment:
nice I like the approach of not refreshing, one nit (and this could be a follow-up or a product discussion) since users are used to the auto-refresh, would it make sense to add a refresh button or something to indicate that getting up-to-date statuses requires manual intervention?

Reply:
+1 to that, maybe a refresh button right next to the checkbox? Or simply a text inviting user to refresh their page?

Reply:
Hmm... i'm open to the idea, but note that interacting with the checkbox will refresh the data. Hard to say since I'm not a UX person whether refresh prompt is needed

Reply:
Oh I did not realize that. In that case that works for me, thanks :)